site/packages/aa-alchemy/signer/overview.md

```diff
@@ -24,10 +24,56 @@ The Alchemy Signer is a `SmartAccountSigner` that is powered by Alchemy's Signer
 
 ## Usage
 
-Once you have been granted access to the Alchemy Signer, getting started is really simple. Install the `@alchemy/aa-alchemy` package and initialize your signer:
+Once you have enabled the Alchemy Signer in the [dashboard](https://dashboard.alchemy.com/accounts?a=account-kit-docs), getting started is really simple. Install the `@alchemy/aa-alchemy` package and initialize your signer:
 
 <<< @/snippets/signers/alchemy/signer.ts
 
+### Using the Signer with Smart Contract Accounts
+
+Once your signer is authenticated with a user, you can use it to sign User Operations by creating a `SmartContractAccount` and passing the signer to it. For example:
+
+::: code-group
+
+```ts
+import { signer } from "./signer";
+
+export const account = await createMultiOwnerModularAccount({
+  transport: rpcTransport,
+  chain,
+  signer,
+});
+```
+
+<<< @/snippets/signers/alchemy/signer.ts
+:::
+
+### Using the Signer as an EOA
+
+::: warning
+Note that EOA wallets will not have access to smart account features like gas sponsorship, batched transactions, multi-owner, or plugins. If you want to switch from EOA to smart accounts later, then each user will need to transfer their assets from the EOA account to a new smart account. It is not currently possible to "upgrade" and EOA to a smart contract account, although the community is discussing potential [EIPs](https://eips.ethereum.org/EIPS/eip-7377) to do that in the future.
+:::
+
+Because the Alchemy Signer has its own `address` and supports signing messages as raw hashes, it is possible to use this signer as an EOA directly. To do so, you can adapt the AlchemySigner to your library of choice and leverage its `signMessage`, `signTypedData`, and `signTransaction` methods directly. The public address of the signer can be accessed via `getAddress`.
```

Comment:
```suggestion
Because the Alchemy Signer has its own `address` and supports signing messages as raw hashes, it is possible to use this signer as an EOA directly. To do so, you can adapt the Alchemy Signer to your library of choice and leverage its `signMessage`, `signTypedData`, and `signTransaction` methods directly. The public address of the signer can be accessed via `getAddress`.
```